setup.py

```diff
@@ -57,10 +57,12 @@
         "isodate ==0.6.*",
         "jsonschema >=3.0.0, ==3.*",
         "networkx >= 2.5, ==2.*",
+        "numpy ==1.*",
         "packaging >=19.2",
         "pandas >=1.0.0, ==1.*",
         "phylo-treetime >=0.9.3, ==0.9.*",
         "pyfastx >=0.8.4, ==0.8.*",
+        "scipy ==1.7.*",
```

Comment:
> Note: The latest version of scipy is 1.10.0, but 1.7.3 is the last version compatible with Python 3.7 which is still supported by Augur.

If we know Augur works with scipy 1.10.0, then we should be able to relax this restriction and let Pip's dependency resolver figure out a suitable scipy version depending on the Python version. e.g. maybe:

```suggestion
        "scipy ==1.*",
```

if we don't have any other lower bound.

Reply:
Augur should work with scipy 1.10.0, but TreeTime doesn't and thus Augur CI fails with a new warning. I just took a deeper look and created this bug: https://github.com/neherlab/treetime/issues/223

My plan:

1. Wait for the warning to removed by a new TreeTime release which addresses the issue above.
2. Bump TreeTime's version in this PR.
3. Apply the suggestion above.
4. Re-open this PR for review.

Reply:
Applied suggested change in a7413a3ed85c7fd8044fc43eefe5e9258b2bc058.